raiden-cli/src/raiden.ts

```diff
@@ -1,6 +1,6 @@
 import { Raiden, RaidenChannel, RaidenChannels, ChannelState } from 'raiden-ts';
 import { LocalStorage } from 'node-localstorage';
-import _ from 'lodash';
+import { flatMap, values, filter } from 'lodash';
```

Comment:
Same. Unfortunatelly, one line for each, but you get used to ;)

Reply:
I got rid of lodash. This file was already deleted anyways :D